src/main/java/net/neoforged/neoforge/common/extensions/ITagBuilderExtension.java

```diff
@@ -20,7 +21,9 @@ default TagBuilder getRawBuilder() {
      * @param tagEntry The tag entry to add to the remove list
      * @param source   The source of the caller for logging purposes (generally a modid)
      * @return The builder for chaining purposes
+     * @deprecated Use {@link TagBuilder#remove(TagEntry)} instead.
      */
+    @Deprecated(forRemoval = true, since = "1.21.1")
     default TagBuilder remove(final TagEntry tagEntry, final String source) {
```

Comment:
Did you intentionally omit the single-arg overload for this method?

Reply:
Yes, because the single-arg overload already exists in the `TagBuilder` class.

Reply:
Ah, makes sense, I should have checked the source😅